### PR TITLE
docs update - Intro, Extractors, Sorters, + others

### PR DESCRIPTION
**Pull request by alejoe91**

### CODE REVIEW

File: examples/contribute/plot_make_spike_sorter.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""
+Implement a spike sorter
+===========================
+
+"""
```
Comment (samuelgarcia) [inline examples/contribute/plot_make_spike_sorter.py]: I think this should be in doc but not in the gallery.

**Comment by samuelgarcia** [comment]
super cool.

My only doubt is that the requirement_rtd.txt link the github install.
The idea of this metapackage is to have a fixed a version set.
So requirement_rtd.txt could be in conflict with requirement.txt (that force pypi).
Could requirement_rtd.txt be the same as requirement.txt ?

So we are sure that setup.py + test in travis + rdt force the same version list (in requirement.txt)

**Comment by samuelgarcia** [comment]
Could have only one travis ?
again the same problem.

**Comment by alejoe91** [comment]
I dont know why but klusta fails to install. I think we can point to a specific release instead of master. Without klusta it's building. Some examples will fail at the moment, but the restshould work (including the API!)

**Comment by alejoe91** [comment]
@samuelgarcia I added you as maintainer to the rtd

**Comment by alejoe91** [comment]
I think I fixed it. Travis fails because the -r is missing.
As soon as the rtd finish building I'll make a commit

**Comment by alejoe91** [comment]
ok now locally it runs perfectly with no errors. 
Timeout in rtd.... :(

**Comment by samuelgarcia** [comment]
If timeout on rtd we will need travais build or circle-ci ...

**Comment by alejoe91** [comment]
We could somehow build it on travis and upload it directly to rtd. Is it possible?

**Comment by alejoe91** [comment]
@samuelgarcia Almost all done! I just wanted to update the description in the first page and the overview.

**Comment by alejoe91** [comment]
Added a test that check that all imports work fine. I had to add requests in the requirements because it's used by the MdaRecordingExtractor2 of spikesorters. @magland we should move that to spikeextractors ASAP

**Comment by samuelgarcia** [comment]
@alejoe91 : tell me when I can merge.

From my side, I did not make so progress. But prting existing doc is very fast.

**Comment by alejoe91** [comment]
I'll make some last changes tomorrow morning and merge!

**Comment by samuelgarcia** [comment]
Can I merge now ?
You will push new change after the merge.
So I can also merge my PR.
So we will have an overview on the doc.

File: examples/modules/extractors/plot_supported_formats.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+# * **MaxOne** - MaxOneRecordingExtractor
+# * **MCSH5** - MCSH5RecordingExtractor
+# * **MEArec** - MEArecRecordingExtractor
+# * **Mountainsort MDA** - MdaRecordingExtractor
+# * **Neurodata Without Borders** - NwbRecordingExtractor
+# * **Open Ephys** - OpenEphysRecordingExtractor
+# * **Phy/Kilosort** - PhyRecordingExtractor/KilosortRecordingExtractor
+# * **SpikeGLX** - SpikeGLXRecordingExtractor
+# * **Spyking Circus** - SpykingCircusRecordingExtractor
+
```
Comment (samuelgarcia) [inline examples/modules/extractors/plot_supported_formats.py]: I don't think having a hard list in example is a good idea...
I would put this somewhere esle in the doc. Not in gallery.

File: requirements_rtd.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+h5py
+tqdm
+click
+klusta
+klustakwik2
+
+# tridesclous
+tridesclous==1.2.1
+
+
```
Comment (samuelgarcia) [inline requirements_rtd.txt]: we can remove this finally I think. RTD will be able to build this.

**Pull request merged**